app/api/users.py

```diff
@@ -43,6 +43,14 @@ class Meta:
     thumbnail = fields.Str()
     small = fields.Str()
     icon = fields.Str()
+    notification = Relationship(attribute='notification',
+                                self_view='v1.user_notification',
```

Comment:
Hey, are you sure you are able to build URL for this? Because I don't see any view with ```user_notification```. If you check marshmallow JSON API docs
```
author = fields.Relationship(
        self_url='/articles/{article_id}/relationships/author',
        self_url_kwargs={'article_id': '<id>'},
        related_url='/authors/{author_id}',
        related_url_kwargs={'author_id': '<author.id>'}
    )
```
You basically need to provide relationship view to```self_view```.